src/Game/Game.php

```diff
@@ -1,4 +1,4 @@
-<?php namespace JoeStrong\RetroAchievements;
+<?php namespace JoeStrong\RetroAchievements\Game;
```

Comment:
PSR rules says that namespace is next line after `<?php`

Reply:
I agree, never seen it on the same line before but I just stayed consistent with what was already there.